tests/test_cases/test_avalon_stream/test_avalon_stream.py

```diff
@@ -55,7 +53,7 @@ def test_avalon_stream(dut):
     tb.backpressure.start(wave())
 
     for _ in range(20):
-        data = random.randint(0,(2^7)-1)
+        data = random.randint(0,(2 ^ 7)-1)
```

Comment:
This is meant to be an exponent, right?

```suggestion
        data = random.randint(0, (2**7)-1)
```

Reply:
Oh, yes, likely; fixed.